src/containers/User/UserArea.tsx

```diff
@@ -51,7 +51,7 @@ const UserArea: React.FC = () => {
 }
 interface MainMenuBarProps {
   templates: TemplateInList[]
-  dataViews: { tableName: string; title: string; code: string }[]
+  dataViews: { tableName: string; title: string; code: string; urlSlug: string }[]
```

Comment:
`urlSlug` is just a kebab-case version of the `code`, which is nicer for browser urls.